wear/src/main/java/io/homeassistant/companion/android/settings/SettingsComposable.kt

```diff
@@ -0,0 +1,161 @@
+import androidx.wear.compose.material.Text
+import androidx.wear.compose.material.ToggleChip
+import androidx.wear.compose.material.ToggleChipDefaults
+import com.mikepenz.iconics.compose.Image
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.home.HomeActivity
+import io.homeassistant.companion.android.home.HomePresenter
+import io.homeassistant.companion.android.util.CommonFunctions
+import io.homeassistant.companion.android.viewModels.EntityViewModel
+
+class SettingsComposable {
+
+    @Composable
+    fun ScreenSettings(swipeDismissableNavController: NavHostController, entityViewModel: EntityViewModel, presenter: HomePresenter) {
+        Column {
+            Spacer(modifier = Modifier.height(20.dp))
+            CommonFunctions().SetTitle(id = R.string.settings)
+            Chip(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 20.dp),
+                icon = {
+                    Image(asset = CommunityMaterial.Icon3.cmd_star)
+                },
+                label = {
+                    Text(
+                        text = stringResource(id = R.string.favorite)
+                    )
+                },
+                onClick = {
+                    swipeDismissableNavController.navigate(
+                        HomeActivity.SCREEN_SET_FAVORITES
+                    )
+                },
+                colors = ChipDefaults.primaryChipColors(
+                    contentColor = Color.Black
+                )
+            )
+            Chip(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 10.dp),
+                icon = {
+                    Image(asset = CommunityMaterial.Icon.cmd_delete)
+                },
+                label = {
+                    Text(
+                        text = stringResource(id = R.string.clear_favorites),
+                    )
+                },
+                onClick = {
+                    entityViewModel.favoriteEntities = mutableSetOf()
+                    CommonFunctions().saveFavorites(entityViewModel.favoriteEntities.toMutableSet(), presenter)
+                },
+                colors = ChipDefaults.primaryChipColors(
+                    contentColor = Color.Black
+                ),
+                secondaryLabel = {
+                    Text(
+                        text = stringResource(id = R.string.irreverisble)
+                    )
+                },
+                enabled = entityViewModel.favoriteEntities.isNotEmpty()
+            )
+        }
+    }
+
+    @Composable
+    fun ScreenSetFavorites(
+        validEntities: List<Entity<Any>>,
+        scalingLazyListState: ScalingLazyListState,

```

Comment:
I believe this should be a separate object. Now the state of the list is shared between lists. This also causes the set favorites list to not start at the top for me.

Reply:
hmm I think that its probably best to always start at the top here, maybe later on we can sort based on the checked status?